docs/developing-prometheus-rules-and-grafana-dashboards.md

```diff
@@ -39,23 +39,55 @@ local kp =
     },
   };
 
-{ 'setup/0namespace-namespace': kp.kubePrometheus.namespace } +
```

Comment:
Likewise the comment on README, this change doesn't seem to be right. 

It makes perfect sense to document the functions you're adding though, maybe you could update the [Changing Default Rules section](https://github.com/prometheus-operator/kube-prometheus/blob/main/docs/developing-prometheus-rules-and-grafana-dashboards.md#changing-default-rules)?

Reply:
This will be moved to the section "Changing Default Rules".